### PR TITLE
Crypto Coin Chart functionality

### PR DESCRIPTION
**Pull request by alejoreyes96**
Added functionality to display a simple chart for a cryptocurrency with the following command:
/chart COIN DAYS
where:
- COIN is the symbol of the cryptocurrency
- DAYS is the span that the chart uses to obtain and plot data

eg. /chart btc 5

### CODE REVIEW

**Comment by ChristianPerez34** [comment]
Closes #14

File: handlers/base.py [hunk 1]
```diff
@@ -29,12 +29,14 @@ async def send_welcome(message: Message):
...
-        f"{bold('/alert')} {italic('COIN')} " "\[< or >] ",
+        f"{bold('/alert')} {italic('COIN')} "
+        "\[< or >] ",
         f"{italic('PRICE')} to set an alert for when the coin reaches set price\n\n",
         f"{bold('/latest')}\_{bold('listings')} to display latest crypto listings\n\n"
         f"{bold('/restart')}\_{bold('kucoin')} to restart KuCoin bot 🤖\n\n",
         f"{bold('/register')} {italic('ADDRESS')} {italic('PRIVATE')}\_{italic('KEY')} to register to use PancakeSwap bot 🤖\n\n",
-        f"{bold('/buy')}\_{bold('coin')} {italic('ADDRESS')} {italic('BNB')}\_{italic('AMOUNT')} to buy coins on pancakeswap",
+        f"{bold('/buy')}\_{bold('coin')} {italic('ADDRESS')} {italic('BNB')}\_{italic('AMOUNT')} to buy coins on pancakeswap\n\n",
+        f"{bold('/chart')} {italic('COIN')} {italic('DAYS_AMOUNT')} to display coin chart\n\n",
```
Comment (ChristianPerez34) [inline handlers/base.py:39]:
```python
f"{bold('/chart')} {italic('COIN')} {italic('DAYS')}\_{italic('AMOUNT')} to display coin chart\n\n",
```

Comment (ChristianPerez34) [reply handlers/base.py:39]: So that help text displays proper styles and avoid text craziness we've seen that I introduced previously.

**Review by ChristianPerez34 — APPROVED** [review]
Ans his name is John Cena 🔊

File: handlers/base.py (hunk 1 above)
Comment (alejoreyes96) [reply handlers/base.py:39]: I used the emojize and text imports that were originally on base.py so that the replies would style out correctly

Comment (ChristianPerez34) [reply handlers/base.py:39]: Say no more

**Pull request merged**